src/util/debug.h

```diff
@@ -112,6 +112,7 @@ int rotate_debug_files(void);
 #define SSSDBG_TRACE_ALL      0x4000   /* level 9 */
 #define SSSDBG_BE_FO          0x8000   /* level 9 */
 #define SSSDBG_TRACE_LDB     0x10000   /* level 10 */
+#define SSSDBG_PERF_STAT     0x20000   /* level 9 */
```

Comment:
I think this bit should be added to `SSSDBG_IMPORTANT_INFO`.

Reply:
done